Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:latest
+
+LABEL maintainer="pacocp5@correo.ugr.es"
+
+RUN apk add --update \
+                    ruby \
+                    make 
+
+RUN adduser -D userRuby
+
+USER userRuby
+
+# Obtenido de la imagen oficial de entorno
+ENV GEM_HOME /home/userRuby/.local/bundle
+ENV BUNDLE_SILENCE_ROOT_WARNING=1 \
+	BUNDLE_APP_CONFIG="$GEM_HOME"
+ENV PATH $GEM_HOME/bin:$PATH
```

Comment:
¿Por qué usas tres sentencias `ENV` seguidas?

Reply:
:arrow_up:

Reply:
Una es ENV y la otra es ENV PATH Si lo pongo todo en una sentencia ENV no funciona porque para actualizar la variable PATH, igualmente si borro el Gemfile de la imagen todas estas variables no son necesarias.

Reply:
`ENV PATH` no es una orden y si dices que es una orden, es que no estás entendiendo lo que estás haciendo; si lo has hecho así porque "no funciona" es, una vez más, porque no estás entendiendo lo que estás haciendo. Trata de entenderlo, y de camino trata de entender que cuando yo pongo un comentario, conviene que tratéis de endender por qué lo hago y luego actual en consecuencia.

Reply:
He buscado cómo poner varias variables de entorno seguidas en una sola sentencia ENV y sinceramente no entiendo el error que me da . Ese fragmento lo extraje de la imagen oficial de Ruby, así que si está así es por algo. Creo que me llevaría más tiempo entender el error (deuda técnica) que ponerlo en dos sentencias separadas, aunque no entienda muy bien el por qué no funciona. 

Si eso no te vale, en una tutoría (al volver de las fiestas) lo vemos, pero me parece absurdo calentarme tanto la cabeza por unas buenas prácticas que ni si quiera la imagen oficial de Ruby sigue. Hago commit con lo que tengo y ya lo vemos.

Reply:
Yo no puedo entender las cosas por ti; si no entiendes el error conviene que lo entiendas y, una vez más, como funciona la orden `ENV`. Mira a ver si ves alguna diferencia entre las órdenes en las líneas 14 y 17 y la orden en las líneas 15 y 16. ¿A qué crees que se puede deber? Para entender la diferencia, o bien lo miras, o bien te consultas el manual de la orden correspondiente; o tratas de entender el error, que para eso son los errores.
Efectivamente, se trata de buenas prácticas. Pero, sobre todo, se trata de entender lo que estamos haciendo y no poner cosas hasta que funcione. En este caso, se trata de entender qué hace `ENV`, entre otras órdenes de un Dockerfile.